hardeneks/cluster_wide/cluster_autoscaling/cluster_autoscaler.py

```diff
@@ -39,184 +37,216 @@ def _get_policy_documents_for_role(role_name, iam_client):
     return actions
 
 
-def check_any_cluster_autoscaler_exists(resources: Resources):
+class check_any_cluster_autoscaler_exists(Rule):
+    _type = "cluster_wide"
```

Comment:
So I really like this method of setting the metadata. I'm wondering, should we take it further and put this into the config? each test we run in the config would have its own set of metadata? Possibly could allow for extensions that users could write on their own. 

Just a thought maybe another iteration we do something like that.

Reply:
definitely. That is where I was going :) This is to make the extension easier.